aerospike_enterprise/metadata.csv

```diff
@@ -0,0 +1,651 @@
+aerospike.namespace.client_read_filtered_out,gauge,,transaction,,The number of client read transactions that did not happen because the record was filtered out via a predicate expression.,0,aerospike,,,
+aerospike.namespace.client_read_filtered_out.count,count,,transaction,,The number of client read transactions that did not happen because the record was filtered out via a predicate expression.,0,aerospike,,,
+aerospike.namespace.client_read_not_found,gauge,,,,The number of client read transaction that resulted in not found.,0,aerospike,,,
+aerospike.namespace.client_read_not_found.count,count,,,,The number of client read transaction that resulted in not found.,0,aerospike,,,
+aerospike.namespace.client_read_success,gauge,,,,The number of successful client read transactions.,0,aerospike,,,
+aerospike.namespace.client_read_success.count,count,,,,The number of successful client read transactions.,0,aerospike,,,
+aerospike.namespace.client_read_timeout,gauge,,,,The number of client read transaction that timed out.,0,aerospike,,,
+aerospike.namespace.client_read_timeout.count,count,,,,The number of client read transaction that timed out.,0,aerospike,,,
+aerospike.namespace.client_tsvc_error,gauge,,,,The number of client transactions that failed in the transaction service before attempting to handle the transaction. ,0,aerospike,,,
+aerospike.namespace.client_tsvc_timeout,gauge,,,,The number of client transactions that timed out while in the transaction service before attempting to handle the transaction.,0,aerospike,,,
+aerospike.namespace.client_udf_complete,gauge,,,,The number of completed udf transactions initiated by the client.,0,aerospike,,,
+aerospike.namespace.client_udf_error,gauge,,,,The number of failed udf transactions initiated by the client.,0,aerospike,,,
+aerospike.namespace.client_udf_error.count,count,,,,The number of failed udf transactions initiated by the client.,0,aerospike,,,
+aerospike.namespace.client_udf_filtered_out,gauge,,transaction,,The number of client udf transactions that did not happen because the record was filtered out via a predicate expression.,0,aerospike,,,
+aerospike.namespace.client_udf_timeout,gauge,,,,The number of client udf class that timedout,0,aerospike,,,
+aerospike.namespace.client_write_error,gauge,,,,The number of client write transactions that failed with an error.,0,aerospike,,,
+aerospike.namespace.client_write_error.count,count,,,,The number of client write transactions that failed with an error.,0,aerospike,,,
+aerospike.namespace.client_write_filtered_out,gauge,,transaction,,The number of client write transactions that did not happen because the record was filtered out via a predicate expression.,0,aerospike,,,
+aerospike.namespace.client_write_filtered_out.count,count,,transaction,,The number of client write transactions that did not happen because the record was filtered out via a predicate expression.,0,aerospike,,,
```

Comment:
```suggestion
aerospike.namespace.client_write_filtered_out,gauge,,transaction,,The number of client write transactions that did not happen because the record was filtered out using a predicate expression.,0,aerospike,,,
aerospike.namespace.client_write_filtered_out.count,count,,transaction,,The number of client write transactions that did not happen because the record was filtered out using a predicate expression.,0,aerospike,,,
```